### ISSUE TITLE
Double output on --help

### ISSUE THREAD

**Issue by jeffutter** [fraserxu/electron-pdf#18]
I started using electron-pdf on version 0.6. It never worked for be because of issue #11 . Now that that is fixed I have noticed the --help output is doubled.

```
~/P/cv ❯❯❯ electron-pdf --help
A command line tool to generate PDF from URL, HTML or Markdown files

Options
  --help                     Show this help
  --version                  Current version of package
  -i | --input               String - The path to the HTML file or url
  -o | --output              String - The path of the output PDF
  -c | --css                 String - The path to custom CSS
  -b | --printBackground     Boolean - Whether to print CSS backgrounds.
                               false - true
  -s | --printSelectionOnly  Boolean - Whether to print selection only
                               false - default
  -l | --landscape           Boolean - true for landscape, false for portrait.
                               false - default
  -m | --marginType          Integer - Specify the type of margins to use
                               0 - default
                               1 - none
                               2 - minimum

Usage
  $ electron-pdf <input> <output>
  $ electron-pdf <input> <output> -l

Examples
  $ electron-pdf http://fraserxu.me ~/Desktop/fraserxu.pdf
  $ electron-pdf ./index.html ~/Desktop/index.pdf
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l -c my-awesome-css.css
A command line tool to generate PDF from URL, HTML or Markdown files

Options
  --help                     Show this help
  --version                  Current version of package
  -i | --input               String - The path to the HTML file or url
  -o | --output              String - The path of the output PDF
  -c | --css                 String - The path to custom CSS
  -b | --printBackground     Boolean - Whether to print CSS backgrounds.
                               false - true
  -s | --printSelectionOnly  Boolean - Whether to print selection only
                               false - default
  -l | --landscape           Boolean - true for landscape, false for portrait.
                               false - default
  -m | --marginType          Integer - Specify the type of margins to use
                               0 - default
                               1 - none
                               2 - minimum

Usage
  $ electron-pdf <input> <output>
  $ electron-pdf <input> <output> -l

Examples
  $ electron-pdf http://fraserxu.me ~/Desktop/fraserxu.pdf
  $ electron-pdf ./index.html ~/Desktop/index.pdf
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l -c my-awesome-css.css
```

**Comment by fraserxu** [comment]
@jeffutter Can you try 0.6.1?

**Comment by jeffutter** [comment]
My mistake. I should have mentioned that I am using 0.6.1 now. Although the electron-pdf --version doesn't output the version number.

```
❯ electron-pdf --version
A command line tool to generate PDF from URL, HTML or Markdown files

Options
  --help                     Show this help
  --version                  Current version of package
  -i | --input               String - The path to the HTML file or url
  -o | --output              String - The path of the output PDF
  -c | --css                 String - The path to custom CSS
  -b | --printBackground     Boolean - Whether to print CSS backgrounds.
                               false - true
  -s | --printSelectionOnly  Boolean - Whether to print selection only
                               false - default
  -l | --landscape           Boolean - true for landscape, false for portrait.
                               false - default
  -m | --marginType          Integer - Specify the type of margins to use
                               0 - default
                               1 - none
                               2 - minimum

Usage
  $ electron-pdf <input> <output>
  $ electron-pdf <input> <output> -l

Examples
  $ electron-pdf http://fraserxu.me ~/Desktop/fraserxu.pdf
  $ electron-pdf ./index.html ~/Desktop/index.pdf
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l
  $ electron-pdf ./README.md ~/Desktop/README.pdf -l -c my-awesome-css.css
```

```
~/P/cv ❯❯❯ npm ls -g |grep electron-pdf
├─┬ electron-pdf@0.6.1
```

**Comment by fraserxu** [comment]
@jeffutter Thanks for the report, fixed in 0.6.2.
